ui:
  max_log_num: 500
  video_width : 640
  video_height : 480

topic:
  color_img_raw : "/camera/color/image_raw"
  depth_img_raw : "/camera/depth/image_raw"
  log_text : "log_text"
  processed_image : "processed_image"

log:
  log_rate : 1

log_level:
  debug : 0
  info : 1
  warning : 2
  error : 3

model:
  input_num : 3
  input_width : 640
  input_height : 480